deepmd/entrypoints/train.py

```diff
@@ -112,9 +125,19 @@ def _do_work(jdata: Dict[str, Any], run_opt: RunOptions):
         reset_default_tf_session_config(cpu_only=True)
 
     # init the model
-    model = DPTrainer(jdata, run_opt=run_opt)
-    rcut = model.model.get_rcut()
-    type_map = model.model.get_type_map()
+    rcut_list = []
+    if not run_opt.multi_task:
+        model = DPTrainer(jdata, run_opt=run_opt)
+        rcut = model.model.get_rcut()
+        type_map = model.model.get_type_map()
+    else:
+        model = DPMultitaskTrainer(jdata, run_opt=run_opt)
+        for model_name in model.model_dict.keys():
+            sub_model = model.model_dict[model_name]
+            rcut_list.append(sub_model.get_rcut())
+            type_map = sub_model.get_type_map()
```

Comment:
Thus, we only use the `type_map` of the last data system? Or it is ensured `type_map` among data systems are the same.

Reply:
they are the same for now